mxnet_model_service.py

```diff
@@ -109,3 +109,22 @@ def _inference(self, data):
         self.mx_model.forward(DataBatch(data))
         return self.mx_model.get_outputs()
 
+    def signature(self):
+        """Return the signature(input, output shape) for the model service. 
+
+        Returns
+        -------
+        Dictionary
+            Signature of the model service
+        """
+        return self.signature
```

Comment:
We can change signature attribute to be internal if we provide a signature property. Change self.signature to self._signature?

Reply:
Changed